omod-2.0/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource2_0.java

```diff
@@ -9,9 +9,9 @@
 import org.openmrs.module.webservices.rest.web.annotation.Resource;
 
 @Resource(name = RestConstants.VERSION_1 + "/attachment", supportedClass = Attachment.class, supportedOpenmrsVersions = {
-        "2.0.*" })
+		"2.0.*" })
 public class AttachmentResource2_0 extends AttachmentResource1_10 {
-	
+
```

Comment:
Always run `mvn clean install` prior to submitting a PR. The Maven build takes care of formatting the Java code.